__test__/server/api/campaign/campaign.test.js

```diff
@@ -840,6 +843,105 @@ describe("Reassignments", () => {
     expect(texterCampaignDataResults2.data.assignment.allContactsCount).toEqual(
       29
     );
+    await runGql(
+      dynamicReassignMutation,
+      {
+        organizationUuid: testCampaign.joinToken,
+        campaignId: testCampaign.id,
+      },
+      testTexterUser2
+    );
+     // TEXTER 1 (60 needsMessage, 4 messaged)
+    // TEXTER 2 (25 needsMessage, 2 needsResponse, 3 convo, 1 messaged)
```

Comment:
Would it make more sense to move these comments to above line 933?

Reply:
Yes, fixed!